src/components/LaunchFilter/types/LaunchesAction.ts

```diff
@@ -0,0 +1,14 @@
+import { Launch } from '../../../shared/api/types/Launch';
+import { LaunchFilters } from '../constants/filerConstants';
+
+export interface FilterAction {
+  type: string;
+  payload: {
+    filterValue: LaunchFilters;
+    error?: string;
+    [LaunchFilters.PREVIOUS]?: Launch[];
+    [LaunchFilters.CREWED]?: Launch[];
+    [LaunchFilters.CALIFORNIA]?: Launch[];
+    [LaunchFilters.FLORIDA]?: Launch[];
+  };
```

Comment:
FilterAction should be split into 3 different interfaces: success, failure etc.

Reply:
done